packages/web3/src/utils/djb2.ts

```diff
@@ -22,4 +22,4 @@ export default function djb2(bytes: Uint8Array): number {
     hash = (hash << 5) + hash + (bytes[`${i}`] & 0xff)
   }
   return hash
-}
+}
```

Comment:
The change of this file is unnecessary.

Reply:
ok

Reply:
I have reverted the changes i made to this file `djb2.ts`